selfdrive/car/subaru/subarucan.py

```diff
@@ -27,12 +27,19 @@ def create_es_distance(packer, es_distance_msg, pcm_cancel_cmd):
 
   return packer.make_can_msg("ES_Distance", 0, values)
 
-def create_es_lkas(packer, es_lkas_msg, visual_alert, left_line, right_line):
+def create_es_lkas(packer, es_lkas_msg, visual_alert, left_line, right_line, left_lane_depart, right_lane_depart):
 
   values = copy.copy(es_lkas_msg)
   if visual_alert == VisualAlert.steerRequired:
     values["Keep_Hands_On_Wheel"] = 1
 
+  if visual_alert == VisualAlert.ldw and values["LKAS_Alert"] == 0:
```

Comment:
why checks LKAS_ALERT?

Reply:
the `LKAS_Alert` field is used by stock to display all dash warnings including collision warnings.  I feel it is best not to over-write with the left/right lane departure if the stock collision warning is wanting to warn the driver about an impending collision.